### PR TITLE
Feature/fetch flow execution updates

### LINKED ISSUE #23: Add Fetch Flow Execution Updates API call
Add the corresponding API call, as [documented in the Azkaban project](https://azkaban.github.io/azkaban/docs/latest/#api-fetch-flow-execution-updates)

`This API call fetches the updated information for an execution. It filters by lastUpdateTime which only returns job information updated afterwards.`

### PR DESCRIPTION
**Pull request by Perkles**
Closes: #23 
Fetches the updated information for an execution. It filters by lastUpdateTime which only returns job information updated afterwards

### CODE REVIEW

**Comment by dmvieira** [comment]
Please, check conflicts

File: azkaban_cli/api.py [hunk 1]
```diff
@@ -499,3 +499,30 @@ def fetch_flow_execution_request(session, host, session_id, exec_id):
...
     return response
+
+def fetch_flow_execution_updates_request(session, host, session_id, exec_id, last_update_time):
+    """Fetch a flow execution updates request for the Azkaban API
+
+    :param session: A session for creating the request
+    :type session: requests.Session
+    :param str host: Hostname where the request should go
+    :param str session_id: An id that the user should have when is logged in
+    :param str exec_id: Execution id to be fetched
```
Comment (aguimaraesduarte) [inline azkaban_cli/api.py:510]: Missing docstring entry for `last_update_time`

File: azkaban_cli/azkaban_cli.py [hunk 2]
```diff
@@ -380,6 +381,32 @@ def __fetch_flow_execution(ctx, execution_id):
     except FetchFlowExecutionError as e:
         logging.error(str(e))
 
+def __log_flow_execution_updates(json):
+    logging.info('Id: %s' % (json.get('id')))
+    logging.info('Start Time: %s' % (json.get('startTime')))
+    logging.info('Attempt: %s' % (json.get('attempt')))
+    logging.info('Status: %s' % (json.get('status')))
+    logging.info('Update Time: %s' % (json.get('updateTime')))
```
Comment (aguimaraesduarte) [inline azkaban_cli/azkaban_cli.py]: Missing `flow` and `endTime` from main response body

File: azkaban_cli/azkaban_cli.py [hunk 3]
```diff
@@ -537,6 +564,14 @@ def fetch_flow_execution(ctx, execution_id):
     """Fetch a flow execution"""
     __fetch_flow_execution(ctx, execution_id)
 
+@click.command(u'fetch_flow_execution_updates')
+@click.pass_context
+@click.argument(u'execution_id', type=click.STRING)
+@click.argument(u'last_update_time', type=click.STRING)
```
Comment (aguimaraesduarte) [inline azkaban_cli/azkaban_cli.py]: Maybe set `-1` to be the default value for `last_update_time`?

**Review by aguimaraesduarte — CHANGES_REQUESTED** [review]

File: azkaban_cli/api.py (hunk 1 above)
Comment (Perkles) [reply azkaban_cli/api.py:510]: Added to **azkaban_cli/api.py** 
````
    :param last_update_time: The criteria to filter by last update time. Set the 
     value to be -1 if all job information are needed. Use -lt="value" to
     subscribe the default value, defaults to -1
    :type last_update_time: str, optional
````

File: azkaban_cli/azkaban_cli.py (hunk 2 above)
Comment (Perkles) [reply azkaban_cli/azkaban_cli.py]: Add to **azkaban_cli/azkaban_cli.py** 
````
    logging.info('Flow: %s' % (json.get('flow')))
    logging.info('Flow end time: %s' % (json.get('endTime')))
````

File: azkaban_cli/azkaban_cli.py (hunk 3 above)
Comment (Perkles) [reply azkaban_cli/azkaban_cli.py]: Add default value and help description to ``last_update_time``
````
 @click.option('-lt', 'last_update_time', type=click.STRING, default=u"-1", help=u'The criteria to filter by last update time', show_default=True)
````

**Review by aguimaraesduarte — APPROVED** [review]

**Comment by aguimaraesduarte** [comment]
Looks good to me, @Perkles ! Thank you!

**Pull request merged**